app/client.py

```diff
@@ -4,13 +4,22 @@
 from selenium.webdriver.support import expected_conditions as EC
 from selenium.webdriver.support.ui import WebDriverWait
 
+def click_javascript_void(xpath, driver):
+    WebDriverWait(driver, 20).until(
+        EC.element_to_be_clickable((By.XPATH, "{0}".format(xpath)))
```

Comment:
format string 형식을  f-string 방식으로 하고 있으니까 수정하시면 좋을 것 같아요
```suggestion
        EC.element_to_be_clickable((By.XPATH, f"{xpath}"))
```
or
```suggestion
        EC.element_to_be_clickable(f"{By.XPATH}{xpath}")
```

Reply:
반영하였습니다!